website/docs/docs/dbt-cloud/dbt-cloud-api/service-tokens.md

```diff
@@ -1,38 +1,90 @@
+Service account tokens enable you to securely authenticate with the dbt Cloud API by assigning each token a narrow set of permissions that more precisely manages access to the API. While similar to [User API tokens](user-tokens), service account tokens belong to an account rather than a user.
 
-### Service Account Admin API tokens
+You can use service account tokens for system-level integrations that do not run on behalf of any one user. Assign any permissions set available in dbt Cloud to your service account token.
 
-<Changelog>
+:::info Enterprise and Team plan differences
+Enterprise plans can apply any permission sets available to service tokens. Team plans can apply three permissions sets to service tokens: Owner, Metadata, or Job Admin.
+:::
 
- - Service Account Admin tokens were launched in February, 2021
+For more on permissions sets, see "[Enterprise Permissions](docs/dbt-cloud/access-control/enterprise-permissions)."
 
-</Changelog>
+## Generating service account tokens
 
-Service Account Admin API tokens currently have full read+write access to an account, so please use them with caution.
+In the Account Settings view of dbt Cloud, you can click on the Service Account tokens page and generate a new token. Create and save your token somewhere safe.
 
-#### Generating a Service Account Admin Token
+:::caution Note
 
-In the Account Settings view of dbt Cloud, you can click on the Service Account tokens page and generate a new Account Admin token.  Create and save your token. Please note that you will not be able to view this token again after its is generated, so store it somewhere safe for later use.
+You will not be able to view this token again after generating it, so store the token somewhere safe for later use.
 
-### Metadata Only Service Account Tokens
+:::
 
-<Changelog>
+## Permissions for service account tokens
 
- - Metadata Only Service Account tokens were launched in April, 2021
+You can assign service account tokens any permission set available in dbt Cloud. When you assign a permission set to a token, you will also be able to choose whether to grant that permissions to all projects in the account or to specific projects.
 
-</Changelog>
+### Service account tokens for Team plans
 
-dbt Cloud users can create metadata only service account based API Tokens to authorize requests to the metadata API.
+The following permissions can be assigned to a service account token on a Team plan.
 
-#### Generating a Metadata Only Service Account Token
+**Account Admin service token:**<br/>
+Account Admin service tokens have full `read + write` access to an account, so please use them with caution.  A Team plan refers to this permission set as an "Owner role."
```

Comment:
Should we link to the Account Admin section from [this page](https://docs.getdbt.com/docs/dbt-cloud/access-control/enterprise-permissions#account-admin) here? Or did we remove that to avoid confusion between Team/Enterprise plans?

Reply:
I did remove it to avoid confusion. @saimaddali should I add it back?